Data/DbInitializer.cs

```diff
@@ -72,12 +72,56 @@ public static void Initialize(IServiceProvider serviceProvider)
               }
               context.SaveChanges();
 
+              var productSubTypes =  new ProductSubType[]
+              {
+                  //1
+                  new ProductSubType {
+                      Label = "Portable Electronics",
+                      ProductTypeId = 1
+                  },
+
+                  //2
+                  new ProductSubType {
+                      Label = "Video Game Systems",
+                      ProductTypeId = 1
+                  },
+
+                  //3
+                  new ProductSubType {
+                      Label = "Kitchen Appliances",
+                      ProductTypeId = 2
+                  },
+
+                  //4
+                  new ProductSubType {
+                      Label = "Yard Equipment",
+                      ProductTypeId = 2
+                  },
+
+                  //5
+                  new ProductSubType {
+                      Label = "Living Room",
+                      ProductTypeId = 3
+                  },
+
+                  //6
+                  new ProductSubType {
+                      Label = "Bedroom",
+                      ProductTypeId = 3
+                  }
+              };
+
+             foreach (ProductSubType i in productSubTypes)
+              {
+                  context.ProductSubType.Add(i);
+              }
 
```

Comment:
There is a `context.saveChanges()` on line 363.  Will this not save everything added to the context at that point?  

Also, line 365 and 366 are duplicates of 362 and 363.  Will remove them.